### PR TITLE
Implement community status component

### PR DESCRIPTION
**Pull request by hochiw**
## Features
- Implemented regular and header type community status component
- Added reuglar community status to overview screen

## Pending
- Loading and error state for overview screen

## Screenshots

| Desktop | Mobile |
|---------|--------|
|Screenshot 2022-05-28 at 12 00 55 PM|Screen Shot 2022-05-28 at 12 01 17|
|Screenshot 2022-05-25 at 7 19 35 PM| - |

based on #59  #61 
refs #50

### CODE REVIEW

**Comment by chihimng** [comment]
any update on the design question for humanizing big numbers?

**Comment by hochiw** [comment]
> any update on the design question for humanizing big numbers?

yes designer agreed on using 10M, 10B with a minified version of the full number at the bottom, i will update later

File: react-app/src/components/OverviewScreen/OverviewScreen.graphql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+query LatestCommunityStatus {
```
Comment (chihimng) [inline react-app/src/components/OverviewScreen/OverviewScreen.graphql:1]: This should be OverviewScreenQuery?

And we should have a OverviewScreenAPI.ts?

Can we also introduce RequestState?

File: react-app/src/components/CommunityStatus/CommunityStatusRegular.tsx [hunk 2]
```diff
@@ -51,6 +52,32 @@ export const CommunityStatusRegular: React.FC<CommunityStatusRegularProps> = (
 ) => {
   const { className, chainInfo, inflation, bondedRatio, communityPool } = props;
 
+  const communityPoolAmount = useMemo(() => {
+    if (!communityPool)
+      return {
+        millified: "0",
+        raw: "0",
+      };
+    // Return normal value if communityPool is less than a million
```
Comment (chihimng) [inline react-app/src/components/CommunityStatus/CommunityStatusRegular.tsx:61]: Refactor this to a util?

**Review by chihimng — CHANGES_REQUESTED** [review]

**Comment by chihimng** [comment]
plz rebase

**Comment by hochiw** [comment]
@chihimng updated :adore2x:

File: react-app/src/components/OverviewScreen/OverviewScreen.graphql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+query OverviewScreenQueryCommunityStatus {
```
Comment (chihimng) [inline react-app/src/components/OverviewScreen/OverviewScreen.graphql:1]: we can name this `OverviewScreenQuery`? if there is anything extra needed we will combine in one query anyways?

Comment (chihimng) [reply react-app/src/components/OverviewScreen/OverviewScreen.graphql:1]: can rename later

**Review by chihimng — APPROVED** [review]

File: react-app/src/utils/number.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import BigNumber from "bignumber.js";
+import millify from "millify";
+
+export function convertToReadableNumber(
```
Comment (chihimng) [inline react-app/src/utils/number.ts:4]: can these be named more straight forward? a bit hard to guess which does what

**Review by chihimng — CHANGES_REQUESTED** [review]

**Comment by hochiw** [comment]
@chihimng updated :adore2x:

**Review by chihimng — APPROVED** [review]

**Pull request merged**